### PR TITLE
feat: add hello-world example for kubebb component

### LINKED ISSUE #128: A hello world

### PR DESCRIPTION
**Pull request by bjwswang**
#### What type of PR is this?
/kind feat

#### Which issue(s) this PR fixes

Fixes #128

### CODE REVIEW

**Comment by Abirdcfly** [comment]
pls remove `charts/hello-world/.DS_Store` file

**Review by Abirdcfly — APPROVED** [review]

**Comment by bjwswang** [comment]
@Carrotzpc pls review this hello-world component.

**Comment by bjwswang** [comment]
@Carrotzpc

File: charts/hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+### Configuration
+
+| Parameter           | Description                  | Default                           |
+|---------------------|------------------------------|-----------------------------------|
+| `ingress.enable`     | Enable ingress or not       |       false                       |
+| `ingress.ingressClassName`  | Ingress Class        | `portal-ingress`                  |
+| `ingress.ingressDomain`     | Ingress Domain       |      172.18.0.2.nip.io            |
+| `image`             | Image of the component-store | `kubebb/hello-world:0.0.1`   |
```
Comment (Carrotzpc) [inline charts/hello-world/README.md:61]: `kubebb/hello-world:0.1.0` ?

**Pull request merged**